examples/amp-video-iframe/consent.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <title>amp-video-iframe</title>
+    <link rel="canonical" href="amps.html" />
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <script
+      async
+      custom-element="amp-consent"
+      src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-video-iframe"
+      src="https://cdn.ampproject.org/v0/amp-video-iframe-0.1.js"
+    ></script>
+    <style amp-custom>
+      body {
+        font-family: sans-serif;
+        max-width: 600px;
+      }
+      [placeholder] {
+        background: #ddd;
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <amp-video-iframe
+      id="myVideo"
+      src="/examples/amp-video-iframe/frame-es2015-consent.html"
+      width="16"
+      height="9"
+      layout="responsive"
+      autoplay
+    >
```

Comment:
I think in most cases, this element would also be blocked by `amp-consent` via the `data-block-on-consent` attribute.

Reply:
I added a note to component docs:

https://github.com/ampproject/amphtml/blob/564ad74d934e2afd85df683fe45a3ac947318f8b/extensions/amp-video-iframe/amp-video-iframe.md#L432

Would that be sufficient?

Reply:
Sorry, what I meant was that if the component is not blocked with `data-block-on-consent`, then `getConsentData` will not return the collect consent information. It needs to be blocked w/ `data-block-on-consent` (it will unblock immediately, if we have stored consent data)